documentation/en/install-lotus-ubuntu.md

```diff
@@ -15,29 +15,42 @@ These steps will install the following dependencies:
 - llvm (proofs build)
 - clang (proofs build)
 
-Run
+Install dependencies
 
 ```sh
 sudo apt update
-sudo apt install mesa-opencl-icd ocl-icd-opencl-dev
+sudo apt install mesa-opencl-icd ocl-icd-opencl-dev gcc git bzr jq pkg-config
 ```
 
-Build
+Install Go 1.14
+
+Find the latest version of Go [on their website](https://golang.org/dl/) and follow the installation instructions. At the time of writing this document, thats 1.14.4. Extract it to `/usr/local`, and add the go binaries to your `$PATH`.
 
 ```sh
-sudo add-apt-repository ppa:longsleep/golang-backports
-sudo apt update
-sudo apt install golang-go gcc git bzr jq pkg-config mesa-opencl-icd ocl-icd-opencl-dev
```

Comment:
What's wrong with this PPA?

Reply:
I see 1.14.2 is in the PPA since 04-09 :sunglasses: I ran this installation a few days before that, so I just missed it and didn't run an update when switching branches, but instead manually installed go 1.14.

I shall revert the change. Thanks!